packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -94,12 +94,14 @@ export const peoplePaneContainerTokens: IStackTokens = {
 /**
  * @private
  */
-export const participantListWrapper = mergeStyles({
-  width: '20rem',
-  // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
-  maxWidth: '50vw',
-  height: '100%'
-});
+export const participantListWrapper = (theme: Theme): string =>
+  mergeStyles({
+    width: '20rem',
+    // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
+    maxWidth: '50vw',
+    height: '100%',
+    boxShadow: theme.effects.elevation16
```

Comment:
This looks like a significant change to make file sharing more responsive, on chat on desktop view does it now have a drop shadow?
The drop shadow appears to be applied the whole way around the participant container, if the composite doesn't span the whole screen can the shadow be seen out the top, bottom and right of the container?